resources/views/community/components/claims/finished-claim-table-row.blade.php

```diff
@@ -5,7 +5,9 @@
 
 $claimSetTypeCopy = ClaimSetType::toString(ClaimSetType::NewSet);
 if ($claim['SetType'] !== ClaimSetType::NewSet) {
-    $claimSetTypeCopy = ClaimSetType::toString(ClaimSetType::Revision);
+    $claimSetTypeCopy = '🗒️ ' . ClaimSetType::toString(ClaimSetType::Revision);
+} else {
+    $claimSetTypeCopy = "💡 <b>$claimSetTypeCopy</b>";
```

Comment:
In https://github.com/RetroAchievements/RAWeb/pull/1503 a lightbulb emoji is used to convey sets that have received revisions with unearned achievements. This lightbulb emoji may conflict with the intent of that one if both changes are merged.